### PR TITLE
add subcompany plan bar

### PR DESCRIPTION
**Pull request by ghislaineguerin**
Changes to plan top bar - staged at https://apps-stage-4.risevision.com/
This requires dev work in order to add the expected behavior as detailed here
https://docs.google.com/document/d/1EV8UnPjUzHGGrHMd_I6KxNOUSfSq2tw5akCWdBl11Rw/edit#heading=h.czrnnptnmid5

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Staged at https://apps-stage-3.risevision.com

@Rise-Vision/apps please review. Thanks!

File: src/locales/locales_po/en/displays-app.po [hunk 1]
```diff
@@ -292,8 +292,11 @@ msgstr "View Terms and Conditions"
 msgid "alerts.webServiceURL"
 msgstr "Web Service URL"
 
-msgid "details.all-licenses-used"
```
Comment (fjvallarino) [inline src/locales/locales_po/en/displays-app.po]: @alex-deaconu is this message not displayed anymore? I think it's a situation that can still happen

Comment (fjvallarino) [reply src/locales/locales_po/en/displays-app.po]: @alex-deaconu dismiss, I see it's a different message in Apps now

Comment (alex-deaconu) [reply src/locales/locales_po/en/displays-app.po]: @fjvallarino that is correct. I had the same concern but I confirmed with @ghislaineguerin on this topic. We are showing a "Get More" link instead (https://github.com/Rise-Vision/rise-vision-apps/pull/755/files#diff-15e79bbd8e228a6fddc2bfa898ac9919R44). Thanks.

**Comment by fjvallarino** [comment]
@alex-deaconu looks good 👍 (I don't have permissions to approve)

**Pull request merged**